README.md

```diff
@@ -32,11 +32,51 @@ iris.each do |record|
      record.petal_width,
      record.class,
   ]
-  # [5.1, 3.5, 1.4, 0.2, "Iris-setosa"]
-  # [7.0, 3.2, 4.7, 1.4, "Iris-versicolor"]
 end
+# => [5.1, 3.5, 1.4, 0.2, "Iris-setosa"]
+# => [4.9, 3.0, 1.4, 0.2, "Iris-setosa"]
+  :
+# => [7.0, 3.2, 4.7, 1.4, "Iris-versicolor"]
 ```
 
+Here is an example to access iris dataset by `Table#to_h`:
+
+```ruby
+require "datasets"
+
+iris = Datasets::Iris.new
+iris_table = iris.to_table.to_h
```

Comment:
`iris_hash` instead of `iris_table` will be a better name.

Reply:
Thank you very much.
Changed.